### ISSUE TITLE
loss=nan

### ISSUE THREAD

**Issue by Zoehanlu** [LuckyDoggy/GeoIE#2]
Why is loss=nan occurring during training?

**Comment by LuckyDoggy** [comment]
Please provide more information

**Comment by Zoehanlu** [comment]
Run GeoIE/training.py according to the code provided by your GitHub. This code shows loss = nan.
![image](https://user-images.githubusercontent.com/31984667/54665113-12a02580-4b21-11e9-8ba0-d5b5cede0598.png)

**Comment by LuckyDoggy** [comment]
I have not met a similar problem.Have you modified the code? You can also email me. My email is  774842817@qq.com

**Comment by xw-wang97** [comment]
@Zoehanlu @LuckyDoggy Could you tell me how to run this code on a new dataset, and what the format of the dataset is?
